include/unifex/get_exception_ptr.hpp

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <unifex/tag_invoke.hpp>
+#include <unifex/type_traits.hpp>
+
+#include <memory>
+
+#include <unifex/detail/prologue.hpp>
+
+namespace unifex
+{
+  inline const struct _get_exception_ptr_cpo {
```

Comment:
I would rename this to simply `_fn` and move it into a nested `unifex::_get_exception_ptr` namespace. That way, we get to keep the `unifex` namespace unpolluted for ADL purposes.